src/sentry/tasks/integrations/github/patch_parsers.py

```diff
@@ -0,0 +1,34 @@
+import re
+from abc import abstractmethod
+from typing import Set
+
+
+class LanguageParser:
```

Comment:
Does this need to be declared as an abstract class?

Reply:
i don't think there's a decorator for an abstract class? each method should have the `@abstractmethod` decorator

Reply:
```
from abc import ABC

class LanguageParser(ABC):
```
I think it'd be like this

Reply:
oh i see. yes i can add this